static/css/components/header-bar.less

```diff
@@ -145,6 +145,15 @@
       height: 44px;
       padding: 0 3px;
     }
+
+
+    #toggle {
+      opacity: 0;
+    }
+
+    #toggle:checked ~ .hamburger-dropdown-component ul {
```

Comment:
You've got the right idea here but the problem is the existing rule in static/css/components/header-bar.less that looks like this:
```
.account-component:hover .navigation-dropdown-component,
.hamburger-component:hover .navigation-dropdown-component, .logo-component a, .logo-component img, .navigation-component li:hover .navigation-dropdown-component
```

You should remove the `.hamburger-component:hover .navigation-dropdown-component, .logo-component a, .logo-component img, ` selector so your changes can shine :)

You also need to make sure when #toggle is not checked the menu is hidden.

e.g. `#toggle ~ .hamburger-dropdown-component ul { display: none; }`